rune/libenclave/internal/runtime/pal/api_linux_v3.go

```diff
@@ -3,6 +3,24 @@ package enclave_runtime_pal // import "github.com/inclavare-containers/rune/libe
 /*
 #include <stdlib.h>
 #include <errno.h>
+#include "skeleton/liberpal-skeleton.h"
+
+static int palGetVersion(void *sym)
```

Comment:
I believe you don't need to re-implement this APU in v3.

Reply:
You Can refer to API v2 which doesn't re-implement this API.

Reply:
Thank for the suggestion.
it seems that there are some confusion on pal layer. 
But I will change as you said.